macos/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+if (UNIX AND APPLE)
```

Comment:
combine these two conditions into one?
Also, should it be valid to set BUILD_MACOS_BUNDLE to true, if `UNIX AND APPLE` is false?

Reply:
I think no. I spitted it for upcoming MacOS additions e.g. dragNdrop support I added with latest commit. I think it is not valid to set it to true if it is not `UNIX AND APPLE`.

Reply:
I think this is not necessary. I think it is clear that `BUILD_MACOS_BUNDLE` has no effect if it is not a Mac.